### ISSUE TITLE
Installation of python based module fails on Windows with latest Python/PIP

### ISSUE THREAD

**Issue by bitonio** [akamai/cli#148]
Hi Akamai CLI fellows,

I am assisting a cli-etp user who is struggling with a Windows box and the latest of Python/PIP. See https://github.com/akamai/cli-etp/issues/3

I have been able to reproduce the issue on a clean Windows 10 box with the latest akamai cli:
```
C:\Akamai>akamai install etp
Attempting to fetch command from https://github.com/akamai/cli-etp.git... ... [OK]
Installing... [==        ]  ERROR[0001] Pip not found in the system. Please verify your setup command=install
 command=install not found. Please verify your setup: pip 22.0.4 from C:\Python\lib\site-packages\pip (python 3.10)
Installing... ... [FAIL]
pip not found. Please verify your setup: pip 22.0.4 from C:\Python\lib\site-packages\pip (python 3.10)
 command=install not found. Please verify your setup: pip 22.0.4 from C:\Python\lib\site-packages\pip (python 3.10)
 ERROR[0001] INSTALL ERROR: Unable to install selected package command=install
Unable to install selected package
```
Not only cli-etp seems affected but also cli-eaa and cli-mfa, all Python based.

**Comment by bitonio** [comment]
Looking at the source code, it looks like Python has change it's output of the `pip --version` command.
Now it is
```
pip 22.0.4 from C:\Python\lib\site-packages\pip (python 3.10)
```

And the regex pattern `pipVersionPattern` as defined in https://github.com/akamai/cli/blob/master/pkg/packages/python.go#L37

is likely failing.

**Comment by robertolopezlopez** [comment]
Hi @bitonio ,

Which CLI version?

Regards,

Roberto

**Comment by robertolopezlopez** [comment]
Important: read [this](https://github.com/akamai/cli/#system-dependencies-for-python-based-packages)

**Comment by bitonio** [comment]
@robertolopezlopez CLI version 1.4.1 on Windows 10
The system has the latest python3, pip and virtualenv
I actually used pip to install virtualenv without any issue, ruling out any certificate issue.

**Comment by robertolopezlopez** [comment]
Could you please link the original issue?

On Apr 4, 2022 22:57, Antoine Drochon ***@***.***> wrote:

@robertolopezlopez<https://urldefense.com/v3/__https://github.com/robertolopezlopez__;!!GjvTz_vk!EEWyn286wYe2qj5q9jwuJpXkUKXN0XqBm2ViKY-RmTNqoy6YVlWI6Qk_F9VHKDM$> CLI version 1.4.1 on Windows 10
The system has the latest python3, pip and virtualenv
I actually used pip to install virtualenv without any issue, ruling out any certificate issue.

-

**Comment by bitonio** [comment]
It is on message #1 of this thread :)
https://github.com/akamai/cli-etp/issues/3

**Comment by robertolopezlopez** [comment]
> It is on the first message of this thread :) [akamai/cli-etp#3](https://github.com/akamai/cli-etp/issues/3)

Ooops sorry, I just went directly to the source code and did not read the message properly :-)

I guess that we will have to refactor that regular expression then. We will release a fix as soon as possible.

In the meanwhile and just as a workaround, I recommend using the extension directly, without the akamai CLI.

**Comment by robertolopezlopez** [comment]
Hi @bitonio ,

We have already a patch in place to fix this issue in our internal repository. It will be released as soon as possible.

In the meanwhile, as workarounds, I can recommend you to either use the python extensions directly (without akamai cli) or make use of python 3.9.

Regards,

Roberto

**Comment by robertolopezlopez** [comment]
Hi @bitonio ,

This  should be fixed in the new 1.4.2 release which we have done today. Thus I will close this issue now. If you happen to have problems again, please reopen it :-)

Regards,

Roberto
